application/templates/application/application-overview.html

```diff
@@ -0,0 +1,140 @@
+{% block app_overview %}
+<div style="height: 90%; margin-bottom: 5%;">
+    {% if selected_app %}
+    <div class="top-header">
+        <div class="right-pane-container">
+            {% if not selected_app.is_draft %}
+            <h5 style="margin: 1em; text-align: center;">
+                <b>Application Form</b>
+            </h5>
+            <table class="table table-striped" style="border: 1px solid black; border-collapse:
+                inherit; width: 75%; margin: 0 auto 2em;">
+                <tbody>
+                    {% for name, value in selected_app.get_fields %}
+                    <tr>
+                        <th scope="row">{{name}}</th>
+                        <td>{{ value }}</td>
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+            {% else %}
+            <form action="{% url 'dashboard:application:draftExistingApplication' selected_app.pk %}"
+                method="post" class="form-signup">
+                {% csrf_token %}
+                <div class="form-row">
+                    <div class="form-label-group col-md-6">
+                        {{ form.first_name }}
+                        <label for={{ form.first_name.id_for_label }}>{{ form.first_name.label }}</label>
+                        {% if form.first_name.help_text %}
+                        <p class="help">{{ form.first_name.help_text|safe }}</p>
+                        {% endif %}
+                    </div>
+                    <div class="form-label-group col-md-6">
+                        {{ form.last_name }}
+                        <label for={{ form.last_name.id_for_label }}>{{ form.last_name.label }}</label>
+                        {% if form.last_name.help_text %}
+                        <p class="help">{{ form.last_name.help_text|safe }}</p>
+                        {% endif %}
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-label-group col-md-6">
+                        {{ form.email_address.errors }}
+                        {{ form.email_address }}
+                        <label for={{ form.email_address.id_for_label }}>{{ form.email_address.label }}</label>
+                        {% if form.email_address.errors %}
+                        {{ form.email_address.errors }}
+                        {% endif %}
+                    </div>
+                    <div class="form-label-group col-md-6">
+                        {{ form.phoneNumber }}
+                        <label for={{ form.phoneNumber.id_for_label }}>{{ form.phoneNumber.label }}</label>
+                        {% if form.phoneNumber.help_text %}
```

Comment:
Maybe include an error field for phonenumber as well, since you changed the model to include the country extension as well (+1). Our user base is not used to inputting +1 before their numbers (a feedback I received from someone from the city), so it might be helpful to indicate the error.